Sources/ViewInspector/Inspector.swift

```diff
@@ -1,18 +1,125 @@
 import SwiftUI
 
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
-public struct Inspector { }
+public struct Inspector {
+
+    /// Removes the "(unknown context at <memory_address>)" portion of a type name.
+    /// Calls to this method are memoized and retained for the lifetime of the program.
+    /// - Parameter typeName: The raw type name. (e.g. ``SomeTypeName.(unknown context at $138b3290c).SomePropertyName``)
+    /// - Returns: The sanitized type name. (e.g. ``SomeTypeName.SomePropertyName``)
+    static func sanitizeNamespace(ofTypeName typeName: String) -> String {
+        var str = typeName
+
+        if let sanitized = sanitizedNamespacesCache[typeName] {
+            return sanitized
+        }
+
+        let range = NSRange(location: 0, length: str.utf16.count)
+        str = sanitizeNamespaceRegex.stringByReplacingMatches(in: str,
+                                                              options: [],
+                                                              range: range,
+                                                              withTemplate: "")
+
+        // For Objective-C classes String(reflecting:) sometimes adds the namespace __C, drop it too
+        str = str.replacingOccurrences(of: "<__C.", with: "<")
+
+        Inspector.sanitizedNamespacesCache[typeName] = str
+
+        return str
+
+    }
+
+    /// Replaces the generic types of a given type name with a string.
+    /// Calls to this method are memoized and retained for the lifetime of the program.
+    /// - Parameters:
+    ///   - typeName: The original type name. (e.g. ``SomeTypeName<SomeGenericTypeParameter>``)
+    ///   - replacement: The string to replace the generic parameters with. (e.g. "<EmptyView>")
+    /// - Returns: The type name with its generic parameters replaced. (e.g. ``SomeTypeName<EmptyView>``)
```

Comment:
```suggestion
    ///   - typeName: The original type name. (e.g. `SomeTypeName<SomeGenericTypeParameter>`)
    ///   - replacement: The string to replace the generic parameters with. (e.g. "<EmptyView>")
    /// - Returns: The type name with its generic parameters replaced. (e.g. `SomeTypeName<EmptyView>`)
```

Nit: See similar comment above for rationale.